src/Header/DownloadStream.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_HEADER_DownloadStream_H_
+#define GROUP4_HEADER_DownloadStream_H_
+
+#include <fstream>
+#include <string>
+
+// read file from local
+class DownloadStream {
+ public:
+  // filename will include file path, set a default chunkSize
+  DownloadStream(std::string filename, unsigned long chunkSize);
+  // if run successfully, it will return 1 else return 0
+  int readChunk(unsigned long pos, std::string* const binary);
```

Comment:
What's the const for?

Reply:
Since using const ref of string pointer is non-intuitive, so I choose const pointer. The address of the string will not change, so I choose to use const.